src/instructlab/configuration.py

```diff
@@ -685,6 +685,29 @@ class _train(BaseModel):
     )
 
 
+class _model_config(BaseModel):
+    """
+    Class representing the configuration for a model.
+
+    Attributes:
+        id (str): Internal ID referring to a particular model from the list.
+        path (str | None): Path to where the model can be found. Can either be a HF reference or local filepath.
+        system_prompt (str | None): The initial message used to prompt the conversation with this model.
+    """
+
+    id: str = Field(
```

Comment:
`default_factory` for these?

Reply:
and `default` for `id` is missing even if None

Reply:
This will be a user-specified field for now, we shouldn't have a default here.